### PR TITLE
Feature/em 1229 refactor to use docker

### PR DESCRIPTION
**Pull request by benouaer**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EM-1229
### Change description ###
remove obsolete JUI code and refactor demo app to use docker
**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[X] No
```

### CODE REVIEW

File: docker-compose-dependencies.yml [hunk 1]
```diff
@@ -2,17 +2,41 @@
...
     image: hmcts/service-auth-provider-app:latest
     environment:
-      - MICROSERVICEKEYS_JUI_WEBAPP=AAAAAAAAAAAAAAAA
-      - MICROSERVICEKEYS_EM_GW=AAAAAAAAAAAAAAAA
-      - MICROSERVICEKEYS_CCD_DATA=AAAAAAAAAAAAAAAA
-      - MICROSERVICEKEYS_CCD_GW=AAAAAAAAAAAAAAAA
-      - MICROSERVICEKEYS_CCD_DEFINITION=AAAAAAAAAAAAAAAA
-      - MICROSERVICEKEYS_COH_COR=AAAAAAAAAAAAAAAA
-      - microserviceKeys_em_npa_app=AAAAAAAAAAAAAAAA
+      - jwtKey=wThK0f0/lh3FlxFcL4xUWDMI5C1J9KyQBgXV4wseh1e5J1uYJIjvTvArHxQDrYoHJ23xFxjHkOnvNbR5dXRoxA==
```
Comment (velimd) [inline docker-compose-dependencies.yml:34]: should this be included

File: package.json [hunk 2]
```diff
@@ -32,7 +31,7 @@
   },
   "dependencies": {
     "@angular-devkit/build-ng-packagr": "^0.12.1",
-    "@angular/cli": "6.1.0",
```
Comment (velimd) [inline package.json:35]: i thought Linus, updated the viewer to be angular 7

File: src/app/document-viewer-wrapper.component.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
+<section [className]="tabStyle(documentTypeToShow)" [id]="documentTypeToShow">
```
Comment (velimd) [inline src/app/document-viewer-wrapper.component.html:1]: did the name change for the file as i believe, this code was here before.

**Review by velimd — APPROVED** [review]

File: src/app/document-viewer-wrapper.component.html (hunk 3 above)
Comment (benouaer) [reply src/app/document-viewer-wrapper.component.html:1]: yes

File: package.json (hunk 2 above)
Comment (benouaer) [reply package.json:35]: not sure

File: docker-compose-dependencies.yml (hunk 1 above)
Comment (benouaer) [reply docker-compose-dependencies.yml:34]: not sure how it's used, but I don't think it's a secret, and it's included in other repos

**Pull request merged**